bin/deploy

```diff
@@ -100,6 +100,8 @@ else
   echo "Release process is complete."
   echo
   echo "Once confirmed changes are verified, you can:"
-  echo "  git push release-${version}"
-  echo "  git push ${git_tag}"
+  echo "  git push upstream release-${version}"
+  echo "  git push upstream ${git_tag}"
```

Comment:
I would be against hard-coding the branch to push to. These are more of a suggestion, as not everyone has an `upstream` branch. Some keep `origin` as `origin` and others `upstream` or another convention.

Reply:
Just documenting that you should be pushing to the Linode org repo should be enough, in my opinion.